src/npctrade.cpp

```diff
@@ -256,6 +256,52 @@ void item_pricing::adjust_values( const double adjust, const faction *fac )
     }
 }
 
+std::vector<item_pricing *> item_pricing::get_contents_rec()
+{
+    std::vector<item_pricing *> tmp;
+
+    if( this->is_container && this->contents.size() ) {
```

Comment:
```suggestion
    if( is_container && !contents.empty() ) {
```
we don't really do the `this->` thing, it's unecessary really. also .empty() is a function, it's a bit more reasonable to use a boolean than an integer as a boolean